udisks/udisksclient.h

```diff
@@ -41,6 +41,12 @@ void                udisks_client_new                (GCancellable        *cance
                                                       gpointer             user_data);
 UDisksClient       *udisks_client_new_finish         (GAsyncResult        *res,
                                                       GError             **error);
+void                udisks_client_new_for_connection (GDBusConnection     *connection,
+                                                      GCancellable        *cancellable,
+                                                      GAsyncReadyCallback  callback,
+                                                      gpointer             user_data);
+UDisksClient       *udisks_client_new_for_connection_finish (GAsyncResult        *res,
+                                                             GError             **error);
```

Comment:
New API needs to be added to `doc/udisks2-sections.txt.in.in`.

Reply:
Done